.storybook/preview.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import { GeistProvider, CssBaseline, Themes } from '@geist-ui/core';
+import { ThemeProvider } from 'next-themes';
+import { backgroundColor } from '../src/design/@generated/themes'; //design/@generated/themes
```

Comment:
Useless comment here. Same info in the import string.

Reply:
Fix